server/routerlicious/packages/services-utils/src/morganLoggerMiddleware.ts

```diff
@@ -45,10 +45,10 @@ export function jsonMorganLoggerMiddleware(
                 additionalProperties = computeAdditionalProperties(tokens, req, res);
             }
             const properties = {
```

Comment:
I am curious as to whether it might be possible to set a default value for all the empty properties at the Lumber level itself (setProperties?). That way, we don't have to pass in a default value for individual properties and we are less likely to run into this situation?

Reply:
I think this will catch all. However it might hide some bad issues. I think in an early phase of GA, dealing with individual error specific seems a good way for us to learn.

If we can create additional debugging log on Lumber engine level during the catch all for us to research and learn, then it is better I believe.

Reply:
I don't think there is an easy way to do that. Lumber and Lumberjack do not know all the expected properties a metric/log might have. Here, we are putting all of those in a `properties` object (line 47). Let's say `tokens.status(req, res)` is `undefined`.  Then, `properties` will not have a `[HttpProperties.status]` field, since `undefined` fields would not be present in the `properties` object. Then, Lumberjack/Lumber would not be aware of those values, having no way of setting individual values at that layer. I prefer to go through with this change now since it's a simple fix and also since we don't know of other metrics/logs with empty values, other than those coming from HTTP requests.

![Screen Shot 2022-08-10 at 10 01 59 AM](https://user-images.githubusercontent.com/41453887/183973329-8dd56ac9-cb3d-46b2-8737-eb0dbc0596ea.png)